data/journal_check/bug_refs.json

```diff
@@ -58,7 +58,7 @@
             "sle-micro": [ "5.0", "5.1" ],
             "opensuse": ["Tumbleweed", "15.2", "15.3"],
             "microos":  ["Tumbleweed"],
-            "sle": ["15-SP2", "15-SP3"]
+            "sle": ["15-SP2", "15-SP3", "12-SP5"]
```

Comment:
As you are touching this part, could you also add 15-SP4 as well, please ?

Reply:
When i finish with the rest i will add separate commit. leaving it open untill then

Reply:
So... What about adding 15-SP4 ?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12947#issuecomment-891852599
but there are still unsolved comments

Reply:
Ok, leave it :) I will do it later so we wont have failures for sle15sp4.

Reply:
i have run VR also for SP3 FYI.

Reply:
ready for review